src/QMCDrivers/DMC/WalkerControlMPI.cpp

```diff
@@ -566,6 +558,7 @@ void WalkerControlMPI::swapWalkersSimple(MCPopulation& pop,
         adjust.bad_walkers.pop_back();
       }
       MCPWalker& spawned_walker = *(pop.spawnWalker());
+      assert(spawned_walker.DataSet.size() == 9440);
```

Comment:
Hard coded number. Add a FIXME comment to highlight it.

Reply:
Already removed in partial PR.